packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/LegacyUpdateManifest.kt

```diff
@@ -105,15 +105,11 @@ class LegacyUpdateManifest private constructor(
         commitTime = Date()
       } else {
         id = UUID.fromString(manifest.getReleaseId())
-        val commitTimeString = manifest.getCommitTime()
-        commitTime = if (commitTimeString != null) {
-          try {
-            UpdatesUtils.parseDateString(commitTimeString)
-          } catch (e: ParseException) {
-            Log.e(TAG, "Could not parse commitTime", e)
-            Date()
-          }
-        } else {
+        commitTime = try {
+          val commitTimeString = manifest.getCommitTime() ?: throw JSONException("missing commitTime")
```

Comment:
ah, I see. We expect commitTime to be non-null only in prod. (`testFromLegacyManifestJson_Production_NoCommitTime` test tests for this). Okay, this looks good.